app/javascript/packs/commentDropdowns.js

```diff
@@ -15,51 +15,99 @@ const handleCopyPermalink = (closeDropdown) => {
 };
 
 const initializeArticlePageDropdowns = () => {
+  // Gather all dropdown triggers for comment options and profile previews
   const dropdownTriggers = document.querySelectorAll(
-    'button[id^=comment-dropdown-trigger]',
+    'button[id^=comment-dropdown-trigger], button[id^=comment-profile-preview-trigger-]',
```

Comment:
question: what is the `^` for?

Reply:
It just tells the selector that the ID needs to _start with_ that text. So this will match `comment-profile-preview-trigger-22` etc. All the trigger buttons need unique IDs so this helps a lot to gather them all up!

Reply:
Didn't know one could use regexp inside a selector. AWESOME!!

Reply:
@rhymes, this means find all elements whose ID attribute starts with `comment-dropdown-trigger`. See the docs on [attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors).

Reply:
@rhymes, you can also do `=$` for ends with. 😎

Reply:
thanks :D